config/quick-start.yaml

```diff
@@ -7461,6 +7461,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: ARGO_DATAFLOW_CLUSTER_NAME
```

Comment:
This file is auto generated from the quick-start directory, you’ll need to edit the source file too

Reply:
I had add the env config at config/manager/manager.yaml file.